bundles/CoreBundle/EventListener/SearchBackendListener.php

```diff
@@ -73,12 +77,8 @@ public function onPreDeleteElement(ElementEventInterface $e)
     public function onPostUpdateElement(ElementEventInterface $e)
     {
         $element = $e->getElement();
-        $searchEntry = Data::getForElement($element);
-        if ($searchEntry instanceof Data && $searchEntry->getId() instanceof Data\Id) {
-            $searchEntry->setDataFromElement($element);
-            $searchEntry->save();
-        } else {
-            $this->onPostAddElement($e);
-        }
+        \Pimcore::getContainer()->get(MessageBusInterface::class)->dispatch(
```

Comment:
you are in an service here and still use Pimcore::getContainer(). Use DI instead.

Reply:
done